### DISCUSSION TITLE
HTTPProxy logs and other debug tools

### DISCUSSION THREAD

**Discussion by priyakr-git** [projectcontour/contour#4578] [Q&A] [answered]
How can I tell that the traffic is passing through the envoy and then getting routed to the service/app? The httpproxy is valid, no errors. The service pod log shows that traffic is coming in to the service/app, but I cannot tell if it is getting routed to the envoy from the LB or if it is directly coming to the service/app. 

Do I exec into the envoy and look at the logs there? Are there specific logs to look in?

**Comment by sunjayBhatia** [comment, marked as answer]
If you tail Envoy pod logs (for the named `envoy` container) if you haven't changed any access logging settings you should see access logs, similar to the example here: https://projectcontour.io/docs/v1.21.0/config/access-logging/
